lib/AST/ASTContext.cpp

```diff
@@ -1774,7 +1774,7 @@ ModuleDecl *ASTContext::getStdlibModule(bool loadIfAbsent) {
   if (loadIfAbsent) {
     auto mutableThis = const_cast<ASTContext*>(this);
     TheStdlibModule =
-      mutableThis->getModule({ std::make_pair(StdlibModuleName, SourceLoc()) });
+      mutableThis->getModule({{ StdlibModuleName, SourceLoc() }});
```

Comment:
Usually, places in the compiler avoid two levels of `{{` because the types become less clear when reading the code, which is probably why we have that explicit `make_pair` there. Could you add an explicit constructor which initializes the fields and calls that instead? (There are 3-4 other places which are similar in having a `{{`)

Reply:
Fixed.